src/pages/UserListing/UsersListingPage.tsx

```diff
@@ -1,12 +1,101 @@
-import { UsersListingPageWrapper } from './UsersListingPage.styled';
-import { UsersTable } from './components/Table/UsersTable';
-import users from '../../data/sampleUsers.json';
+import { Stack, Box, Typography, CircularProgress } from '@mui/material';
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { IPagination } from '../../types/pagination';
+import { CustomTable } from '../../components/table/CustomTable';
+import { useTranslation } from 'react-i18next';
+import useGetUsersTableColumns from './useGetUsersTableColumns';
+import { ListingPageWrapper } from '../BookingsList';
```

Comment:
ListingPageWrapper seems to be a reused component. But it is placed in BookingsList folder.